### DISCUSSION TITLE
[IntelliSense] No Autocomplete

### DISCUSSION THREAD

**Discussion by fengluo2** [tailwindlabs/tailwindcss#11977] [Help]
**What version of VS Code are you using?**

v1.81.1

**What version of Tailwind CSS IntelliSense are you using?**

v0.10.0

**What version of Tailwind CSS are you using?**

v3.3.3

**What package manager are you using?**

npm

**What operating system are you using?**

Windows

**Tailwind config**

```ts
import type { Config } from 'tailwindcss'

const config: Config = {
  content: [
    './src/pages/**/*.{js,ts,jsx,tsx,mdx}',
    './src/components/**/*.{js,ts,jsx,tsx,mdx}',
    './src/app/**/*.{js,ts,jsx,tsx,mdx}',
  ],
  theme: {
    extend: {
      backgroundImage: {
        'gradient-radial': 'radial-gradient(var(--tw-gradient-stops))',
        'gradient-conic':
          'conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))',
      },
    },
  },
  plugins: [],
}
export default config
```

**VS Code settings**

```json
{
    "translation.baidu.appId": "########",
    "translation.baidu.appKey": "########",
    "editor.detectIndentation": false,
    "explorer.sortOrder": "type",
    "editor.tabSize": 2,
    "javascript.updateImportsOnFileMove.enabled": "always",
    "workbench.editorAssociations": {
      "*.docx": "default"
    },
    "git.suggestSmartCommit": false,
    "files.autoSave": "afterDelay"
}
```

**Reproduction URL**

[github](https://github.com/fengluo2/test1)

**Describe your issue**

No Autocomplete.

**Comment by ItsEthra** [comment]
Your LSP logs might help with troubleshooting, you could try using commonjs syntax, i.e. change `export default` to `module.exports = { /* config */ }`, remove config type import and change config file extension to `.cjs`. That's what helped me at least.

**Comment by bradlc** [comment]
Hey @fengluo2, can you share the logs from the output panel? You can use the `Tailwind CSS: Show Output` command to reveal it.
